index.html

```diff
@@ -24,24 +36,24 @@ <h2>The Past</h2>
                 <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
             </div>
         </section>
-        
+
         <section class="middle-content">
-        
+
             <h2>Why Did It Have To Be Boxes...</h2>
-            
+
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box b1">Box 1</div>
+                <div class="box b2">Box 2</div>
+                <div class="box b3">Box 3</div>
+                <div class="box b4">Box 4</div>
+                <div class="box b5">Box 5</div>
+                <div class="box b6">Box 6</div>
+                <div class="box b7">Box 7</div>
+                <div class="box b8">Box 8</div>
+                <div class="box b9">Box 9</div>
+                <div class="box b10">Box 10</div>
```

Comment:
Ndawi great use of various classes in the same html tag to apply shared styling to all boxed and specific to each one of them